index.html

```diff
@@ -191,22 +191,20 @@ <h2 class="our-partner-title">Our Partners</h2>
         </section>
     </main>
 
-    <section>
-        <!--FOOTER -->
-    </section>
-
-
-</body>
-
-</html>
-
-</main>
-
-<section>
-    <!--FOOTER -->
+    <section class="footer">
+      <div class="container">
+       <div class="box">
+     <h3 class="demo">Hack Your Future (part of Open Knowledge Belgium)</h3>
+    <p class="place">Open Knowledge Belgium vzw
+      Cantersteen 12
+      1000 Bruxelles</p>
+      <a class="place1" href="mailto:contact@hackyourfuture.be">contact@hackyourfuture.be</a> 
+    </div>
+    </div>
 </section>
 
+ </main>
```

Comment:
Can you delete </main>? We have this for a while and didn't notice it was written twice.

Reply:
i removed it
thanks